docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Create a Tile
+type: documentation
+description: Learn how to develop and publish an integration tile.
+aliases:
+- 'developers/marketplace/offering'
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/create_a_tile.md"
+further_reading:
+- link: "https://partners.datadoghq.com/"
+  tag: "Partner Network"
+  text: "Datadog Partner Network"
+- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
+  tag: "Blog"
+  text: "Expand your monitoring reach with the Datadog Marketplace"
+- link: "/developers/integrations/marketplace_offering/"
+  tag: "Documentation"
+  text: "Learn how to build a Marketplace offering"
+- link: "/developers/integrations/oauth_for_integrations"
+  tag: "Documentation"
+  text: "Learn about using OAuth for integrations"
+---
+
+## Overview
+
+This page walks you through how to develop an offering that will be displayed in a tile on the Datadog Integrations page or on the Datadog Marketplace. For all offerings that do not use the Datadog Agent, including API-based integrations, professional services listings, and software licenses, a tile-only listing is required. This means that there is no Agent code attached to the tile, but it serves as an informative point of entry where customers can learn more about your offering, learn how to set it up, as well as install or purchase your offering to unlock out-of-the-box dashboards and other assets. You can see an example of a tile below:
+
+{{< img src="developers/marketplace/marketplace-tile-example.png" alt="Example Marketplace tile" style="width:30%" >}}
+
+**Select an option below to get started and create a tile on either the Marketplace or Integrations page:** 
+
+{{< tabs >}}
+{{% tab "Build a tile on the Marketplace" %}}
+
+## Set up a directory and clone the Marketplace repository
+
+Set up a directory:
+
+1. Request access to the [Marketplace repository][101] by following the instructions in the [Marketplace documentation][102].
+2. Create a `dd` directory:
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+3. Once you have been granted access to the Marketplace repository, create the `dd` directory and clone the `marketplace` repo:
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/marketplace.git{{< /code-block >}}
+4. Create a feature branch to work in.
```

Comment:
Looks odd in the preview
<img width="372" alt="Screen Shot 2023-05-01 at 5 38 40 PM" src="https://user-images.githubusercontent.com/109224841/235536184-50d6b114-0716-4ecb-8511-791831eedf54.png">